### DISCUSSION TITLE
[Question] What are the alternatives to the deprecated Feeds?

### DISCUSSION THREAD

**Discussion by GWTorstenDEV** [amzn/selling-partner-api-models#668] [Q&A] [answered]
Hello,

the following Feeds are marked on March 18, 2024 deprecated and will be removed on March 31, 2025.

https://github.com/amzn/selling-partner-api-docs/discussions/3790
https://developer-docs.amazon.com/sp-api/docs/sp-api-deprecations

What are the alternatives to this Feeds to Upload it via Feeds API?

- `POST_PRODUCT_DATA` 
- `POST_INVENTORY_AVAILABILITY_DATA`
- `POST_PRODUCT_OVERRIDES_DATA`
- `POST_PRODUCT_PRICING_DATA` 
- `POST_PRODUCT_IMAGE_DATA` 
- `POST_PRODUCT_RELATIONSHIP_DATA`    
- `POST_FLAT_FILE_INVLOADER_DATA`
- `POST_FLAT_FILE_LISTINGS_DATA`
- `POST_FLAT_FILE_BOOKLOADER_DATA`
- `POST_FLAT_FILE_CONVERGENCE_LISTINGS_DATA`
- `POST_FLAT_FILE_LISTINGS_DATA`
- `POST_FLAT_FILE_PRICEANDQUANTITYONLY_UPDATE_DATA`
- `POST_UIEE_BOOKLOADER_DATA`

thx in advance

Greets
Torsten

**Comment by dhuhn-sellvin** [comment, marked as answer]
Hello Torsten,

It seems that the Feeds will be replaced by Listing Items API.
You can found a migration guide here: https://developer-docs.amazon.com/sp-api/docs/listing-workflow-migration-tutorial

Not all but the most feeds can be found there. It's sad, that Amazon didn't link from the deprecation message to the tutorial.

Kind regards,
Dennis
